mergify_engine/mergify_pull.py

```diff
@@ -168,6 +169,42 @@ def _get_checks(self):
                                for s in self._get_statuses()])
         return generic_checks
 
+    def _get_team_members(self, name):
+        organization, _, team_slug = name.partition("/")
+        if organization[0] != "@" or not team_slug or '/' in team_slug:
+            # Not a team slug
+            return
+
+        try:
+            g_organization = self.g.get_organization(organization[1:])
+            for team in g_organization.get_teams():
+                if team.slug == team_slug:
+                    return [m.login for m in team.get_members()]
+        except github.GithubException as e:
+            if e.status == 400:
+                return
+            elif e.status >= 500:
+                raise
+            self.log.warning("fail to get the organization, team or members",
+                             team=name, status=e.status,
+                             detail=e.data["message"])
+
+    def resolve_teams(self, values):
+        if not values:
+            return []
+        if not isinstance(values, (list, tuple)):
+            values = [values]
+
+        gh_logins = []
+        for value in values:
+            team_members = self._get_team_members(value)
```

Comment:
if you change `_get_team_members` as suggest you can just do
gh_logins.extend(self._resolve_login(value))

Reply:
done.